### ISSUE TITLE
Cleanup Vendor directories

### ISSUE THREAD

**Issue by jeffmccune** [holos-run/holos#245]
When rendering the platform, if one of the sub-processes fails, the other processes don't clean up their vendor directory properly.  A signal handler is likely necessary for the sub processes.

Noticed this lying around.

```
projects/platform/components/istio/istiod/vendor4022138867/
```

**Comment by jeffmccune** [comment]
This is in [func cacheChart(](https://github.com/holos-run/holos/blob/a44ebe5171f312beb2ed404d1291c7b05b85507b/internal/builder/v1alpha4/builder.go#L406)

**Comment by jeffmccune** [comment]
Should be fixed by #275
